audmetric/core/api.py

```diff
@@ -102,8 +102,8 @@ def concordance_cc(
     """
     assert_equal_length(truth, prediction)
 
-    prediction = np.array(prediction)
-    truth = np.array(truth)
+    prediction = np.array(list(prediction))
+    truth = np.array(list(truth))
```

Comment:
Does it make sense to define a common pre-processing function that applies these to all incoming truth and prediction values in all functions, or can it also break stuff?
At the moment you are doing this fix only for `concordance_cc()` and `pearson_cc()`, and interestingly for all the output of per class in `unweighted_average_fscore()`, `unweighted_average_precision()`, `unweighted_average_recall()`,

Reply:
OK, then let's stay with the current approach for now. At least it is tested, refactoring of the code we could do another time if needed.